src/sql/migrations/20220304184711_schema.js

```diff
@@ -41,7 +41,7 @@ const setupEnums = async (knex) => {
   await knex.raw('CREATE TYPE sample_technology AS ENUM (\'10x\', \'rhapsody\');');
   await knex.raw('CREATE TYPE sample_file_type AS ENUM (\'features10x\', \'barcodes10x\', \'matrix10x\', \'rhapsody\');');
   await knex.raw(
-    'CREATE TYPE upload_status AS ENUM (\'uploaded\', \'uploading\', \'compressing\', \'uploadError\', \'fileNotFound\', \'fileReadError\', \'fileReadAborted\');',
+    'CREATE TYPE upload_status AS ENUM (\'invalidFormat\', \'uploaded\', \'uploading\', \'compressing\', \'uploadError\', \'fileNotFound\', \'fileReadError\', \'fileReadAborted\');',
```

Comment:
I don't think we can do this change here, you need to add a new migration to add `invalidFormat` to the `upload_status`